torchao/prototype/mx_formats/mx_ops.py

```diff
@@ -35,10 +38,12 @@
     tensor_size_hpx3_to_fp6x4,
 )
 from torchao.prototype.mx_formats.utils import to_blocked
+from torchao.utils import fill_defaults
 
 aten = torch.ops.aten
 
 MX_OPS_TABLE: Dict[Any, Any] = {}
+MX_FUNCTION_TABLE: Dict[Any, Any] = {}
```

Comment:
is this needed?

Reply:
So it isn't explicitly needed today but TBH this is the "correct" way to structure subclass dispatching since these two op tables should be treated separately